toqito/states/trine.py

```diff
@@ -21,11 +21,12 @@ def trine() -> list[np.ndarray]:
 
     >>> from toqito.states import trine
     >>>
-    >>> trine()
+    >>> trine()  # doctest: +ELLIPSIS, +NORMALIZE_WHITESPACE
     [array([[1],
-       [0]]), array([[-0.5      ],
-       [-0.8660254]]), array([[-0.5      ],
-       [ 0.8660254]])]
+            [0]]), array([[-0.5      ],
+            [-0.8660254]]), array([[-0.5      ],
+            [ 0.8660254]])]
+
```

Comment:
I do not know how to pass this test without the doctest flags. Expected vs what doctest got are identical. 

![image](https://github.com/vprusso/toqito/assets/66048318/3ad45c33-494e-49cf-83b5-a94fa827a0bb)

Reply:
It's difficult to know from just the screenshot, but I actually think there is an additional space from the number to the end bracket (you can see how the line outputs are slightly off).

I think you if you copy/paste the "expected" output and put that in the comment, that should fix it (I think).